pcmpl-args.el

```diff
@@ -197,16 +197,14 @@ If the exit status is non-zero, an error is signaled."
     retcode))
 
 (defun pcmpl-args-process-lines (program &rest args)
-  "Same as `process-lines' but with logging using `pcmpl-args-debug'."
+  "PROGRAM and ARGS are the same as `process-lines'.
+Logging is enabled if variable `pcmpl-args-debug' is NON NIL."
   (pcmpl-args-debug "!process-lines: %S %S" program args)
   (apply 'process-lines program args))
 
 (defun pcmpl-args-unbackspace-string (string)
   "Remove ^H characters from STRING."
-  (with-temp-buffer
-    (insert string)
-    (call-process-region (point-min) (point-max) "col" t t nil "-b")
-    (string-trim (buffer-string))))
+  (replace-regexp-in-string ".\b" "" string))
```

Comment:
Does this have the same behavior as before? The new command is replace two characters (the backspace and the character before it).

Reply:
Yes, I looked at the source code of `col`